show-vars.md

```diff
@@ -0,0 +1,111 @@
+---
+title: SHOW (session settings)
+summary: The SHOW statement displays the current settings for the client session.
+toc: false
+---
+
+The `SHOW` [statement](sql-statements.html) can display the value of either one or all of
+the session setting variables.
+
+<div id="toc"></div>
+
+## Required Privileges
+
+No [privileges](privileges.html) are required to display the session settings.
+
+## Synopsis
+
+General syntax:
+
+~~~
+   SHOW <variable name>
+   SHOW ALL
+~~~
+
+The variable name is case insensitive.
+It may be enclosed in double quotes; this is useful if the variable name itself contains spaces.
+
+Special syntax cases supported for compatibility:
+
+| Syntax | Equivalent to |
+|--------|---------------|
+| `SHOW TRANSACTION PRIORITY` | `SHOW "transaction priority" |
+| `SHOW TRANSACTION ISOLATION LEVEL` | `SHOW "transaction isolation level" |
+| `SHOW TIME ZONE` | `SHOW "time zone" |
+| `SHOW TRANSACTION STATUS` | `SHOW "transaction status" |
+
+Note: the `SHOW` statement for session settings in unrelated to the
```

Comment:
Wrap in note formatting `{{site.data.alerts.callout_info}}{{site.data.alerts.end}}` and maybe shorten to provide only one example of the type of statement you're referring to.